Need to look into this ci error.

```
=================================== FAILURES ===================================
________________________ test_remove_invalid_connection ________________________

    def test_remove_invalid_connection():
        addresses = [("127.0.0.1", 9669), ("127.0.0.1", 9670), ("127.0.0.1", 9671)]
        configs = Config()
        configs.min_connection_pool_size = 30
        configs.max_connection_pool_size = 45
        pool = ConnectionPool()
    
        try:
            assert pool.init(addresses, configs)
    
            # turn down one server('127.0.0.1', 9669) so the connection to it is invalid
            os.system("docker stop tests_graphd0_1")
            time.sleep(3)
    
            # get connection from the pool, we should be able to still get 30 connections even though one server is down
            for i in range(0, 30):
                conn = pool.get_connection()
                assert conn is not None
    
            # total connection should still be 30
            assert pool.connects() == 30
    
            # the number of connections to the down server should be 0
>           assert len(pool._connections[addresses[0]]) == 0
E           assert 10 == 0
E            +  where 10 = len(deque([<nebula3.gclient.net.Connection.Connection object at 0x7f4debc84700>, <nebula3.gclient.net.Connection.Connection object at 0x7f4dec88cf10>, <nebula3.gclient.net.Connection.Connection object at 0x7f4dec880a30>, <nebula3.gclient.net.Connection.Connection object at 0x7f4decd3c190>, <nebula3.gclient.net.Connection.Connection object at 0x7f4decd54df0>, <nebula3.gclient.net.Connection.Connection object at 0x7f4decd54400>, ...]))

tests/test_pool.py:298: AssertionError
```

--- a/tests/test_pool.py
+++ b/tests/test_pool.py
@@ -283,7 +283,7 @@ def test_remove_invalid_connection():
         assert pool.init(addresses, configs)
 
         # turn down one server('127.0.0.1', 9669) so the connection to it is invalid
-        os.system("docker stop tests_graphd0_1")
+        os.system("docker stop tests_graphd0_1 || docker stop tests-graphd0-1")
         time.sleep(3)
 
         # get connection from the pool, we should be able to still get 30 connections even though one server is down
@@ -302,5 +302,5 @@ def test_remove_invalid_connection():
         assert len(pool._connections[addresses[2]]) == 15
 
     finally:
-        os.system("docker start tests_graphd0_1")
+        os.system("docker start tests_graphd0_1 || docker start tests-graphd0-1")
         time.sleep(3)

--- a/tests/test_session.py
+++ b/tests/test_session.py
@@ -50,8 +50,12 @@ class TestSession(TestCase):
             time.sleep(10)
             for i in range(0, 5):
                 if i == 3:
-                    os.system("docker stop tests_graphd0_1 || docker stop tests-graphd0-1")
-                    os.system("docker stop tests_graphd1_1 || docker stop tests-graphd1-1")
+                    os.system(
+                        "docker stop tests_graphd0_1 || docker stop tests-graphd0-1"
+                    )
+                    os.system(
+                        "docker stop tests_graphd1_1 || docker stop tests-graphd1-1"
+                    )
                     time.sleep(3)
                 resp = session.execute("SHOW SESSIONS")
                 assert resp.is_succeeded(), resp.error_msg()

--- a/tests/test_session.py
+++ b/tests/test_session.py
@@ -50,8 +50,8 @@ class TestSession(TestCase):
             time.sleep(10)
             for i in range(0, 5):
                 if i == 3:
-                    os.system("docker stop tests_graphd0_1")
-                    os.system("docker stop tests_graphd1_1")
+                    os.system("docker stop tests_graphd0_1 || docker stop tests-graphd0-1")
+                    os.system("docker stop tests_graphd1_1 || docker stop tests-graphd1-1")
                     time.sleep(3)
                 resp = session.execute("SHOW SESSIONS")
                 assert resp.is_succeeded(), resp.error_msg()
@@ -63,8 +63,8 @@ class TestSession(TestCase):
         except Exception as e:
             assert False, e
         finally:
-            os.system("docker start tests_graphd0_1")
-            os.system("docker start tests_graphd1_1")
+            os.system("docker start tests_graphd0_1 || docker start tests-graphd0-1")
+            os.system("docker start tests_graphd1_1 || docker start tests-graphd1-1")
             time.sleep(2)
 
     def test_3_session_context(self):

